webapp/package.json

```diff
@@ -61,6 +62,7 @@
     "query-string": "^5.1.0",
     "raf": "3.4.0",
     "react": "16.2.0",
+    "react-circular-progressbar": "^1.0.0",
```

Comment:
I don't see where this is used?

Reply:
uh this is going to be used by the new UI (for the confirmations progressbar)